packages/hooks/src/useEventListener.ts

```diff
@@ -1,9 +1,9 @@
 import { useEffect, useRef } from 'react';
 
 export interface UseEventOptions {
-  options?: AddEventListenerOptions;
+  options?: Omit<AddEventListenerOptions, 'once'>;
   dependencies?: Array<any>;
-  enabled?: boolean;
+  enabled?: boolean | 'once';
```

Comment:
Should we handle `true` and `false` as strings as well?

Reply:
I would want to disallow it in the types and documentation but handle them in the implementation in case non-TS codebases make a mistake. thoughts?

Reply:
I dig that, though I'm a tad worried that it'll be confusing 😬 I'll leave it to you. If we do string "true"/"false", I like your approach.